docs/GRAPHQL.md

```diff
@@ -131,137 +134,6 @@ Serves a GraphQL "query IDE" that can be used for testing out writing queries
 and seeing the responses served for those queries by `POST /api/v2.2/graphql`.
 Note the `/*` in the path - to open this in your browser while running using
 `run.sh`/`smoketest.sh`, go to `https://localhost:8181/api/v2.2/graphiql/`. The
-trailing slash is significant.
-
-## GraphQL API
-
-### Quick Reference
-
-| What you want to do                                                       | Which handler you should use                                                    |
-| ------------------------------------------------------------------------- | --------------------------------------------------------------------------------|
-| **Recordings in Target JVMs**                                             |                                                                                 |
-| Update metadata for a recording in a target JVM                    | [`PutActiveRecordingMetadataMutator`](#PutActiveRecordingMetadataMutator) |
-| **Recordings in archive**                                                 |                                                                                 |
-| Update metadata for an archived recording                          | [`PutArchivedRecordingMetadataMutator`](#PutArchivedRecordingMetadataMutator)     |
-
-### Recordings in Target JVMs
```

Comment:
Should these types of graphql endpoint querying docs be put somewhere else or are we just removing them altogether?

Reply:
I'm removing them altogether. I don't think it's worth maintaining them. End users who want to interact with the GraphQL API should either use the GraphiQL "IDE" or read the schemas. Hand-written docs like our HTTP_API.md are prone to falling out of date, but the schema is always a correct source of information.